requirements.txt

```diff
@@ -1,6 +1,6 @@
 aiofiles==0.6.0
 fastapi==0.65.2
-monai[nibabel, skimage, pillow, tensorboard, gdown, ignite, torchvision, itk, tqdm, lmdb, psutil, openslide]>=0.6.0
+monai-weekly[nibabel, skimage, pillow, tensorboard, gdown, ignite, torchvision, itk, tqdm, lmdb, psutil, openslide]
```

Comment:
Why monai weekly?

Reply:
only early testing.. end of next we rollback to 0.7 stable version when monai is released